.editorconfig

```diff
@@ -16,8 +16,8 @@ indent_size = 2
 [*.md]
 trim_trailing_whitespace = false
 
-# Indentation override for all JS under lib directory
-[wp-content/themes/core/js/src/**/*.js]
+# Indentation override for all JS under assets directory
+[wp-content/themes/core/assets/js/src/**/*.js]
```

Comment:
I vote for changing this to `wp-content/themes/core/**/*.js`, since we may have JS in the `components` and `integrations` directories, too.

Reply:
💯 on what Jonathan said and we do indeed